srcpkgs/snes9x-gtk/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'snes9x-gtk'
+pkgname=snes9x-gtk
+version=1.56.2
+revision=1
+wrksrc=snes9x-$version/gtk/
+build_style=gnu-configure
+hostmakedepends="automake pkg-config"
+makedepends="gettext-devel glib-devel gtk+3-devel intltool minizip-devel SDL2-devel xorg-server-devel"
```

Comment:
move `intltool` to the `hostmakedepends` instead.

Reply:
Thx is now corrected

Reply:
````
nocross="https://travis-ci.org/void-linux/void-packages/jobs/400627809#L973
 https://travis-ci.org/void-linux/void-packages/jobs/400627810#L974"
````